### PR TITLE
create lbwsg exposure component

### PR DESCRIPTION
**Pull request by rmudambi**
### Description

- *Category*: implementation
- *JIRA issue*: [MIC-2993](https://jira.ihme.washington.edu/browse/MIC-2993)
- *Research reference*: [LBWSG documentation](https://vivarium-research.readthedocs.io/en/latest/gbd2019_models/risk_exposures/low_birthweight_short_gestation/index.html#risk-exposure-lbwsg)

Created a LBWSG component which does the following
 - creates a pipeline for generating a pregnancy duration
 - creates a pipeline for generating the future birth weight of a child

Additionally created a function to get a birth date from a conception date and a pregnancy duration.

I needed to update the version of VPH in order to run, and this required the commenting out of some observers, since they are broken due to the VPH observer updates.

### Verification and Testing
Tested in an interactive simulation.

### CODE REVIEW

File: src/vivarium_gates_iv_iron/components/__init__.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-    MortalityObserver,
-    DisabilityObserver,
-    PregnancyObserver,
-    MaternalDisordersObserver,
-    MaternalHemorrhageObserver,
-    HemoglobinObserver,
-    AnemiaObserver,
-)
+from vivarium_gates_iv_iron.components.child import LBWSGExposure
+# from vivarium_gates_iv_iron.components.observers import (
```
Comment (mattkappel) [inline src/vivarium_gates_iv_iron/components/__init__.py:2]: Left over from testing?

**Review by mattkappel — APPROVED** [review]
Not sure if you're planning to leave the observers commented out until the final merge to `develop`. If so, looks fine to me.

File: src/vivarium_gates_iv_iron/components/child.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    def _pregnancy_duration_pipeline_source(self, index: pd.Index) -> pd.Series:
+        pregnancy_duration_propensity = self.randomness.get_draw(
+            index, additional_key='pregnancy_duration'
+        )
+        categorical_propensity = self.randomness.get_draw(index, additional_key='categorical')
+        pregnancy_duration = self.lbwsg_exposure_distribution.single_axis_ppf(
+            GESTATIONAL_AGE, pregnancy_duration_propensity, categorical_propensity
+        )
+        return pregnancy_duration
```
Comment (albrja) [inline src/vivarium_gates_iv_iron/components/child.py:96]: Is this pregnancy_duration going to take the place of the pregnancy duration constant we had set in the pregnancy component?

Comment (rmudambi) [reply src/vivarium_gates_iv_iron/components/child.py:96]: Yes, that's the plan

**Review by albrja — APPROVED** [review]

**Pull request merged**